__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -151,6 +248,200 @@ describe('start', () => {
     wallet.setGapLimit(originalGapLimit);
     hWallet.stop();
   });
+
+
+  it('should start a wallet to manage a specific token', async () => {
+    const walletData = precalculationHelpers.test.getPrecalculatedWallet();
+
+    // Creating a new wallet with a known set of words just to generate the custom token
+    let hWallet = await generateWalletHelper({
+      seed: walletData.words,
+      preCalculatedAddresses: walletData.addresses,
+    });
+    await GenesisWalletHelper.injectFunds(hWallet.getAddressAtIndex(0), 2);
+    const { hash: tokenUid } = await createTokenHelper(
+      hWallet,
+      'Dedicated Wallet Token',
+      'DWT',
+      100
+    );
+
+    // Stopping this wallet and destroying its memory state
+    await hWallet.stop({ cleanStorage: true });
+    hWallet = null;
+
+    // Starting a new wallet re-using the same words, this time with a specific wallet token
+    hWallet = await generateWalletHelper({
+      seed: walletData.words,
+      preCalculatedAddresses: walletData.addresses,
+      tokenUid,
+    });
+    expect(hWallet.isReady()).toStrictEqual(true); // This operation should work
+
+    // Now testing the methods that use this set tokenUid information
+    // FIXME: No need to explicitly pass the non-boolean `false` as a tokenUid to get this result.
```

Comment:
I think the issue #397 on the `getBalance` method will require enough change on the method's code that this could also be done there.
Do you agree with just increasing its scope instead of creating a new issue?